.github/workflows/auto-update-dependencies.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         cargo install cargo-edit
     
     - name: Check for dependency updates
-      uses: romoh/dependencies-autoupdate@v1.1
+      uses: romoh/dependencies-autoupdate@v1.2
```

Comment:
(not for this PR) would it make sense to move dependencies-autoupdate to project-akri?

Reply:
It is a generic github action, so doesn't have to necessary live under akri. 
Maybe we can create a separate project for any generic tooling we create and maintain .